### ISSUE TITLE
esc key patch

### ISSUE THREAD

**Issue by eliasyior** [bypassiwastaken/skiovox-helper#110]
esc key is patched where you can not press esc key when it asks you to press sign out on the first pop up. personally my admin can still sign me out even though in the exploit. I am looking for ways to get passed this problem I figured out that you can still press ctrl+alt+z for chromvox spoken feedback and the magnifying glass and o+t for command reference but when I do that the screen turns black temporarily and then turns white and I haven't weighted long enough to see if it will work so mabey you can do something with at info.

**Comment by eliasyior** [comment]
btw my version of chrom is 121

**Comment by O1n0n** [comment]
im with the same situation too!!!1

**Comment by eliasyior** [comment]
kind of cool honestly I found out that if you do the chromevox spoken feedback gets time it will bring you to a wifi dino screen but that's really it.

**Comment by eliasyior** [comment]
@3kh0 do you know a way to get passed this problem.
if it helps the steps I use are turn of wifi do the shift alt s thing then press the settings button from the night time shader arrow then after that the Ctrl alt z search t o thing then wait till I get into the email sign in part to turn of chrome vox do the sign in btw forgot to tell you I pressed the sign in as a existing user option but from their when it gives me the option to sign out the esc key does not work.

should I just accept that the skiovox method has been patched?
just thought that your really good at this kind of stuff. honestly don't know how you find this stuff out in the first place.

**Comment by eliasyior** [comment]
so in settings here 
![image](https://github.com/bypassiwastaken/skiovox-helper/assets/114923766/30c1ab1e-d757-4032-a5aa-182f1a1d0020)
the night time shade is hear
![Screenshot 2024-02-20 12 15 25 PM](https://github.com/bypassiwastaken/skiovox-helper/assets/114923766/89e7ab36-fa3f-4f3f-9527-25c20009ef2d)
then yoyu press the arrow key new to it at the right and then once you do that you look at the top right corner with the gear and press that button.

**Comment by O1n0n** [comment]
oh damn thats how it looks? cause my stupid school doesnt even update my computer (which kinda sucks)

**Comment by eliasyior** [comment]
darn well can you send a picture please?

**Comment by O1n0n** [comment]
[Screen recording 2024-02-21 7.56.03 AM.webm](https://github.com/bypassiwastaken/skiovox-helper/assets/153309252/bc87d158-54ad-48c3-9555-ab0744cff5e3)
(screen recording)

**Comment by O1n0n** [comment]
cuz my computer is trash

**Comment by eliasyior** [comment]
well what if you press the corner arrow during an attempt to get into skiovox helper just do it when the wifi screen pops up.

**Comment by O1n0n** [comment]
the one near the volume button?

**Comment by eliasyior** [comment]
yes and when you press that button send a picture by pressing Ctrl and alt and the box with the the lings to the right side of it at the top, then you can press and drag the space you want captured.

**Comment by eliasyior** [comment]
also so cool that at least one person cares.

**Comment by eliasyior** [comment]
so thanks.

**Comment by eliasyior** [comment]
btw if you need help with any thing else just comment on this because I keep coming back to check and I might know a thing or two or not.

**Comment by O1n0n** [comment]
got it, btw thanks for being helpful

**Comment by eliasyior** [comment]
yup

**Comment by O1n0n** [comment]
dang my alt shift s got patched

**Comment by eliasyior** [comment]
mine did not. guess thats it for you and skiovox.

**Comment by eliasyior** [comment]
they blocked mine too.

**Comment by CGRmags** [comment]
downgrade to 120 then use caub

**Comment by CGRmags** [comment]
in the skiovox window it works

**Comment by O1n0n** [comment]
for me that would be upgrading since my chrome version is 114.

**Comment by CGRmags** [comment]
WHAT 114? wth

**Comment by O1n0n** [comment]
our computers are trash like i said before

**Comment by eliasyior** [comment]
huh really? and how do I downgrade?

**Comment by O1n0n** [comment]
plus how do i upgrade??

**Comment by PlanetSat0rn** [comment]
im on 118 and i have this issue

I'm planning on downgrading to see if it works.

**Comment by PlanetSat0rn** [comment]
> how to i downgrade

caub

**Comment by eliasyior** [comment]
cool thank you

**Comment by dt5093** [comment]
what is caub

**Comment by JackWagon885** [comment]
CAUB prevents auto updates. Also, having a lower version school Chromebook is great because you have access to more exploits, such as Ingot & the original LTMEAT.

**Comment by dt5093** [comment]
A: how do i downgrade.
B: how do i get it

**Comment by JackWagon885** [comment]
Although I'm making a school website about this stuff, I haven't looked into CAUB because I didn't realize its potential. You can find instructions on many bypass methods on the 3hk0 github page.

**Comment by dt5093** [comment]
Network state is blocked

**Comment by JackWagon885** [comment]
https://github.com/3kh0/ext-remover?tab=readme-ov-file#caub-prevent-updates

**Comment by dt5093** [comment]
I don’t have access to chrome://network#state

**Comment by JackWagon885** [comment]
flag version exists, scroll down

**Comment by rolexwatch123** [comment]
> CAUB prevents auto updates. Also, having a lower version school Chromebook is great because you have access to more exploits, such as Ingot & the original LTMEAT.

somehow mine updated to 121 even though i had caub on i was on 119

**Comment by rolexwatch123** [comment]
> downgrade to 120 then use caub

how do you downgrade

**Comment by JackWagon885** [comment]
Maybe you didn't do it correctly, or you swapped ti an unCAUBed WiFI network. You can dowgrade via these instructions straight off my website:

Requires storage device; extension access required

Downgrade your Chromebook.

1: Go to chrome://version & check your board (under PLatform).

2: Go to chrome100.dev & find your board (use Ctrl+F).

3: Download the version of Chrome OS you want.

4: Install the [Chromebook Recovery Utility](https://chromewebstore.google.com/detail/chromebook-recovery-utili/pocpnlppkickgojjlmhdmidojbmbodfm) extension on another computer.

5: Plug in the storage device & follow the instructions.

6: On your Chromebook, enter Recovery Mode (Esc+Reload+Power) & follow the prompts.

7: Skip the "Checking for Updates" screen by pressing Ctrl+Shift+E.

**Comment by JackWagon885** [comment]
wait what

Either way, 115 is the latest stable version, so what are you doing on 121?

**Comment by JackWagon885** [comment]
After some research, it's not patched.

**Comment by rolexwatch123** [comment]
> wait what
> 
> Either way, 115 is the latest stable version, so what are you doing on 121?

I was on 119 using caub but all of a sudden my Chromebook updated

**Comment by JackWagon885** [comment]
Do you have multiple WiFi networks available to your Chromebook? CAUBing only affects the WiFI networks you select.

**Comment by JackWagon885** [comment]
You can still downgrade & reCAUB

**Comment by rolexwatch123** [comment]
can i use a microsd card?

**Comment by eliasyior** [comment]
I am so glad this got more popular.

**Comment by JackWagon885** [comment]
As long as you can connect it to the Chromebook, you can use it (though 8+ GB is required for Chrome OS images).

**Comment by rolexwatch123** [comment]
i have 120+ gb on my micro sd so im good

**Comment by JackWagon885** [comment]
yea just get it to be read by the Chromebook

**Comment by PlanetSat0rn** [comment]
damn i guess i have some merciful school administrators if all they do is make me unable to skiovox

**Comment by JackWagon885** [comment]
but they can't unless they remove kiosk apps

**Comment by PlanetSat0rn** [comment]
funny story: i can't shift+alt+s nor chromevox

**Comment by JackWagon885** [comment]
h u h
